src/app/room/[slug]/page.module.css

```
@@ -0,0 +1,189 @@
+  );
+
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  width: min(
+    calc(var(--panel-content-width) + (var(--panel-horizontal-padding) * 2)),
+    calc(100vw - 32px)
+  );
+
+  height: 98vh;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)